DropInDemo/Demo Base/Settings/BraintreeDemoSettings.m

```diff
@@ -49,6 +51,11 @@ + (BraintreeDemoTransactionServiceThreeDSecureRequiredStatus)threeDSecureRequire
     return [[NSUserDefaults standardUserDefaults] integerForKey:BraintreeDemoSettingsThreeDSecureRequiredDefaultsKey];
 }
 
++ (BraintreeDemoTransactionServiceThreeDSecureRequestedVersion)threeDSecureRequiredVersion {
```

Comment:
I think Susan point it out earlier, but just commenting here to be safe. The `threeDSecureRequiredVersion` seems like it should be `requested` to me